### PR TITLE
[Teach-428] Update styles on Diversity Survey

### PR DESCRIPTION
**Pull request by levadadenys**
### [Teach-428] Update styles on Diversity Survey

Before:
Знімок екрана 2023-04-27 о 22 31 22

After:
Знімок екрана 2023-04-27 о 22 41 50

## Links
- jira ticket: [Teach-428](https://codedotorg.atlassian.net/browse/TEACH-428?atlOrigin=eyJpIjoiNzYzM2UxNDA4ZTg1NDVjYzhlMTU4ODdlYmZlNWM5ZWYiLCJwIjoiaiJ9)

## PR Checklist:

- [ ] Tests provide adequate coverage
- [ ] Privacy and Security impacts have been assessed
- [ ] Code is well-commented
- [ ] New features are translatable or updates will not break translations
- [ ] Relevant documentation has been added or updated
- [ ] User impact is well-understood and desirable
- [ ] Pull Request is labeled appropriately
- [ ] Follow-up work items (including potential tech debt) are tracked and linked

### CODE REVIEW

File: dashboard/app/views/home/_survey_diversity.haml [hunk 1]
```diff
@@ -26,7 +26,7 @@
 
       #submit
         %button.btn-primary#submitSurvey{type: "submit"} Submit
-        %span#noanswer{onclick: "return submitSurvey(false)"} Prefer not to answer
+        %button#noanswer{onclick: "return submitSurvey(false)"} Prefer not to answer
```
Comment (bethanyaconnor) [inline dashboard/app/views/home/_survey_diversity.haml:29]: Thank you for updating this!

File: dashboard/app/assets/stylesheets/application.scss [hunk 2]
```diff
@@ -3507,6 +3507,18 @@ code {
 
     #noanswer {
```
Comment (bethanyaconnor) [inline dashboard/app/assets/stylesheets/application.scss:3508]: Do you know if there are other elements with ids `noanswer` and `survey`. Unfortunately, they're pretty generic ids...

**Review by bethanyaconnor — APPROVED** [review]
Thanks! One question to double check for any unintended consequences but otherwise looks good!

**Review by hannahbergam — COMMENTED** [review]
🎉

File: dashboard/app/assets/stylesheets/application.scss (hunk 2 above)
Comment (levadadenys) [reply dashboard/app/assets/stylesheets/application.scss:3508]: Nice catch, thanks!
Agree, those are not very specific IDs.
Luckally, we only have them for this specific survey.
+ `#noanswer` styles are set specifically to be aplied if `#noanswer` is a child of `#diversity_survey`. So until we add another `#diversity_survey`  for some reason 😄,  everything will be fine.


Знімок екрана 2023-04-28 о 00 11 12

Comment (levadadenys) [reply dashboard/app/assets/stylesheets/application.scss:3508]: Безіменний

Comment (levadadenys) [reply dashboard/app/assets/stylesheets/application.scss:3508]: image

Comment (bethanyaconnor) [reply dashboard/app/assets/stylesheets/application.scss:3508]: Thanks for checking and all the screenshots!

**Pull request merged**